tekton/ci-images.yaml

```diff
@@ -23,5 +23,18 @@ spec:
     - /kaniko/executor
     args:
     - --dockerfile=/workspace/source/tekton/ko/Dockerfile
-    - --destination=$(inputs.params.imageRegistry)/$(outputs.resources.builtKoImage.url)
-    - --context=/workspace/source
\ No newline at end of file
+    - --destination=${inputs.params.imageRegistry}/${outputs.resources.builtKoImage.url}
+    - --context=/workspace/source
+
+    volumeMounts:
+      - name: gcp-secret
+        mountPath: /secret
+    env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /secret/release.json
+
+  volumes:
```

Comment:
This *could* be specified in the `PipelineRun` with `PodTemplate` (that said it's not supported yet by `tkn`…)